static_frame/core/batch.py

```diff
@@ -951,6 +953,455 @@ def shift(self,
                 fill_value=fill_value,
                 )
 
+    # ---------------------------------------------------------------------------
+    # na handling
+    def isna(self) -> "Batch":
+        """
+        Return a batch of same-indexed, Boolean Frames indicating True which values are NaN or None.
```

Comment:
Here and in other doc-strings, please do this:

```suggestion
        Return a :obj:`Batch` of same-indexed, Boolean :obj:`Frame` indicating True which values are NaN or None.
```